kotlinpoet/src/commonMain/kotlin/com/squareup/kotlinpoet/CodeWriter.kt

```diff
@@ -735,9 +737,10 @@ internal class CodeWriter constructor(
       generatedImports: MutableMap<String, Import>,
       canonicalName: T.() -> String,
       capitalizeAliases: Boolean,
+      forcedAliases: Set<String>,
```

Comment:
`forcedAliases` is not a very precise name IMO, since the set doesn't actually include aliases. My suggestion would be to rename it to `referencedNames`, as that's what we're passing in. By the way I also don't feel like `referencedNames` is a great name, I'll think of a more descriptive one and rename it later, along with this new parameter.

Reply:
Renamed as you suggested